step.yml

```diff
@@ -136,7 +136,15 @@ inputs:
       description: |-
         Specify the output pattern with golang template: [https://golang.org/pkg/text/template](https://golang.org/pkg/text/template)
       is_required: true
-      is_expand: false                    
+      is_expand: false
+  - permanent_download_url_map_format: "{{range $index, $element := .}}{{if $index}}|{{end}}{{$element.File}}=>{{$element.URL}}{{end}}"
+    opts:
+      title: Format for the BITRISE_PERMANENT_DOWNLOAD_URL_MAP output
+      summary: You can change the format of the BITRISE_PERMANENT_DOWNLOAD_URL_MAP output
+      description: |-
+        Specify the output pattern with golang template: [https://golang.org/pkg/text/template](https://golang.org/pkg/text/template)
```

Comment:
Customize the output pattern with this golang template: [https://golang.org/pkg/text/template](https://golang.org/pkg/text/template)

Reply:
The referenced link is rather a template language, not a template itself. Using it you can create templates.

Reply:
Customize the output pattern by using this golang template language: https://golang.org/pkg/text/template